src/azure-cli-core/azure/cli/core/parser.py

```diff
@@ -152,6 +152,9 @@ def error(self, message):
         with CommandLoggerContext(logger):
             logger.error('%(prog)s: error: %(message)s', args)
         self.print_usage(sys.stderr)
+        # Manual recommendations
+        self._set_manual_recommendations(**args)
```

Comment:
Avoid using `**args` when passing variables around. https://medium.com/better-programming/stop-abusing-args-and-kwargs-in-python-560ce6645e14

Reply:
I am open for the parameters, while I prefer kwargs because the function may be extended to handle other possible errors, and may need other source info for action recommendations.